routes/index.js

```diff
@@ -7,4 +7,15 @@ router.get('/', function (req, res, next) {
   res.render('index', { products: products });
 });
 
+router.get('/fiveisprime', function (req, res, next) {
+  res.render('index', { products: [{
+    "name": "Single red garden gnome",
+    "price": 56,
+    "brand": {
+        "name": "Home & Pro tools"
+    },
+    "stockUnits": 98
+}] });
```

Comment:
This formatting looks off. Did you run Prettier?

Reply:
Nope.